mwdb/resources/user.py

```diff
@@ -427,3 +428,36 @@ def put(self, login):
 
         schema = UserSuccessResponseSchema()
         return schema.dump({"login": login})
+
+
+class UserProfileResource(Resource):
+    @requires_authorization
+    def get(self, login):
+        """
+        ---
+        summary: Get profile information
+        description: |
+            Returns information about specific user
+        security:
+            - bearerAuth: []
+        tags:
+            - auth
+        responses:
+            200:
+                description: Returns information about specific user
+                content:
+                  application/json:
+                    schema: UserProfileResponseSchema
+        """
+        schema = UserProfileResponseSchema()
+        if g.auth_user.has_rights(Capabilities.manage_users):
+            user = db.session.query(User).filter(User.login == login).first()
+        else:
+            user = (db.session.query(User)
+                              .join(User.groups)
+                              .filter(Group.name != "public")
+                              .filter(g.auth_user.is_member(Group.id))
+                              .filter(User.login == login)).first()
+        if user is None:
+            raise NotFound("User doesn't exist or you don't have capability to see his profile")
```

Comment:
That one should be a bit more friendly, most users doesn't know about "additional capabilities".
```suggestion
            raise NotFound("User doesn't exist or is not a member of your group")
```